utils.c

```diff
@@ -617,3 +620,34 @@ pivot_root (const char * new_root, const char * put_old)
   return -1;
 #endif
 }
+
+char * label_mount(const char *opt, const char *mount_label) {
+  if (!mount_label)
+    return strdup_printf(opt);
+
+#ifdef HAVE_SELINUX
+  if (!opt)
+    return strdup_printf ("context=\"%s\"", mount_label);
```

Comment:
Well user could give us a bogus label, but the mount command would fail.

Labels for SELinux always look like user_u:object_r:filetype_t:s0:c1,c2